app/mailers/user_mailer.rb

```diff
@@ -0,0 +1,6 @@
+class UserMailer < ActionMailer::Base
+  default from: "admin@meetmeeples.com"
+
+  def password_reset(user)
```

Comment:
Unused method argument - `user`. If it's necessary, use `_` or `_user` as an argument name to indicate that it won't be used. You can also write as `password_reset(*)` if you want the method to accept any arguments but don't care about them.